### PR TITLE
Upgrade/600/logical switch

### PR DESCRIPTION
**Pull request by jyothis-gireesan**
### Description
Added support for logical support for V600

### Check List
- [X] New functionality includes testing.
  - [X] All tests pass (`$ rake test`).
- [X] New functionality has been documented in the README if applicable.
  - [X] New functionality has been thoroughly documented in the examples (please include helpful comments).
- [X] New endpoints supported are updated in the endpoints-support.md file.
- [X] Changes are documented in the CHANGELOG.

### CODE REVIEW

File: examples/shared_samples/logical_switch.rb [hunk 1]
```diff
@@ -57,45 +58,47 @@
 # This section illustrates scope usage with the switch. Supported in API 300 and onwards.
```
Comment (soodpr) [inline examples/shared_samples/logical_switch.rb]: Do correct the comment given here- its supported till 500

File: spec/unit/resource/api600/c7000/logical_switch_spec.rb [hunk 2]
```diff
@@ -1,10 +1,19 @@
...
-  it 'inherits from OneviewSDK::API300::C7000::LogicalSwitch' do
-    expect(described_class).to be < OneviewSDK::API300::C7000::LogicalSwitch
+  it 'inherits from OneviewSDK::API500::C7000::LogicalSwitch' do
+    expect(described_class).to be < OneviewSDK::API500::C7000::LogicalSwitch
+  end
+
+  describe '#initialize' do
+    context 'OneView 4.0' do
+      it 'sets the defaults correctly' do
+        logical_switch = OneviewSDK::API600::C7000::LogicalSwitch.new(@client_300)
```
Comment (soodpr) [inline spec/unit/resource/api600/c7000/logical_switch_spec.rb]: correction: @client_600

File: examples/shared_samples/logical_switch.rb [hunk 3]
```diff
@@ -57,45 +58,47 @@
...
-  puts "Listing logical_switch['scopeUris']:"
-  puts logical_switch['scopeUris'].to_s
-
-  # Delete the scopes
-  scope_1.delete
-  scope_2.delete
-rescue NoMethodError
-  puts "\nScope operations is not supported in this version."
+if @client.api_version >= 300 && @client.api_version <= 500
+  puts "\nOperations with scope."
```
Comment (madhav-bharadwaj) [inline examples/shared_samples/logical_switch.rb]: I think this operation is supported in api600 also. Have a look at API docs to confirm.

Comment (jyothis-gireesan) [reply examples/shared_samples/logical_switch.rb]: In the documentation, it doesn't mention scopes removed. When I tried the URL it was giving the PATCH is not available at this URL.

Comment (madhav-bharadwaj) [reply examples/shared_samples/logical_switch.rb]: Ok. Then you might need to change the endpoint support file to 'heavy minus' for patch. Maybe after confirming with the product dev team.

File: examples/shared_samples/logical_switch.rb [hunk 4]
```diff
@@ -15,12 +15,13 @@
...
 #
 # Supported APIs:
-# - 200, 300, 500 for C7000 only.
+# - 200, 300, 500, 600 for C7000 only.
 
 # Resources that can be created according to parameters:
 # api_version = 200 & variant = any to OneviewSDK::API200::LogicalSwitch
 # api_version = 300 & variant = C7000 to OneviewSDK::API300::C7000::LogicalSwitch
 # api_version = 500 & variant = C7000 to OneviewSDK::API500::C7000::LogicalSwitch
+# api_version = 600 & variant = C7000 to OneviewSDK::API500::C7000::LogicalSwitch
```
Comment (sijeesh) [inline examples/shared_samples/logical_switch.rb]: update comment

**Pull request merged**